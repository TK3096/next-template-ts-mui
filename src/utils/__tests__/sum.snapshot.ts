import sum from '../sum'

it('test function sum', () => {
  const snapshot = {
    value: sum(1, 2),
  }

  expect(snapshot).toMatchSnapshot({
    value: 3,
  })
})
