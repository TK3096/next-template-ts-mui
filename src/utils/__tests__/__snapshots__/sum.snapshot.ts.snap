// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test function sum 1`] = `
{
  "value": 3,
}
`;
